src/tests/Common/Directory.Build.targets

```diff
@@ -167,6 +170,11 @@
       <RunTimeDependencyCopyLocal Include="$(CoreDisToolsLibrary)" Condition="$(CopyCoreDisToolsToCoreRoot)" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'mono' and '$(IsDesktopArchitecture)'" >
+      <RuntimeDependencyCopyLocal Include="$(MonoArtifactsPath)/libcoreclr$(LibSuffix)" TargetDir=""  />
```

Comment:
@naricc I think we also want `libmono-component-*`

Reply:
Hmm, I ran all the JIT tests with out it and they completed successfully. Maybe I should try the whole suite to be sure.

Reply:
@naricc Right now we build the runtime components as static libs on desktop hosts (at least on OSX that's what I see).  That's not totally right, IMO.  But that's why the tests are passing.  In the future we will make them shared libraries, I think.  So right now it's just future-proofing the infrastructure.

Reply:
Thanks, I didn't realize that might be in the works. I may as well future proof it.